changelogs/fragments/165-write-file-sanitize.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - Allow write_file to sanitize spaces and newlines. (https://github.com/ansible-community/antsibull-docs/issues/305)
```

```suggestion
  - Allow write_file to sanitize spaces and newlines (https://github.com/ansible-community/antsibull-docs/issues/305).
```

is the format we generally follow.